fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -686,6 +686,142 @@ countries:
                 length: 3.0
                 issue_months: [8, 7, 6]
                 start_year: 1982
+    southern-oromia:
+        logo: Ethiopia_IRI_98x48.png
+        resolution: [.25, .25]
+        center: [40.0, 5.7]
+        zoom: 6
+        rotation: 0
+        marker:  [40.0, 5.7]
+        origin: [.125, .125]
+        onload_warning: *warn
+        shapes:
+            - name: Project Region
+              sql: select id_1 as key, 'Southern Oromia' as label, ST_AsBinary(ST_Union(the_geom)) as the_geom
+                  from iridb.eth_zones_dd
+                  where name_2 in ('East Shewa', 'Arsi', 'West Arsi', 'West Haraghe', 'East Haraghe', 'Bale', 'East Bale', 'Guji', 'Borena')
+                  group by 1, 2
+              vuln_sql: select cast(null as int) as key, 0 as year, 0 as vuln where 1 = 2
```

Comment:
What does `cast(null as int)` do?

Reply:
in SQL, nulls are typed, but they're typed as strings by default, so this gives you (as you might expect) an int-typed null. If you don't put this cast in it throws an error. 

That being said, I'm kinda thinking we should modify retrieve_vuln such that if there's no vuln_sql field it just doesn't do the join. Having to copy past this same code everywhere is annoying and confusing